packages/account/src/providers/transaction-summary/operations.ts

```diff
@@ -252,7 +257,12 @@ export function getContractCallOperations({
     if (contractInput) {
       const newCallOps = contractCallReceipts.reduce((prevContractCallOps, receipt) => {
         if (receipt.to === contractInput.contractID) {
-          const input = getInputFromAssetId(inputs, receipt.assetId);
+          let input;
+          if (receipt.assetId !== ZeroBytes32) {
+            input = getRelevantInputs(inputs, receipt.assetId);
+          } else {
+            input = getRelevantInputs(inputs);
+          }
```

Comment:
what is the purpose of this conditional, if the method `getRelevantInputs` will anyway use ZeroBytes32 as default value?

Reply:
7004319a6